aeron-client/src/main/java/io/aeron/Aeron.java

```diff
@@ -1902,68 +1886,5 @@ private void connectToDriver()
                 toDriverBuffer = ringBuffer;
             }
         }
-
-        @SuppressWarnings("try")
-        private static MappedByteBuffer waitForFileMapping(
-            final File file, final EpochClock clock, final long deadlineMs)
-        {
-            while (true)
-            {
-                while (!file.exists() || file.length() < CncFileDescriptor.META_DATA_LENGTH)
-                {
-                    if (clock.time() > deadlineMs)
```

Comment:
While it is unlikely to have a millisecond timestamps go negative, I wonder if it is good style/practice for us to use the form:
```
if (clock.time() - deadlineMs < 0)
```